pkg/api/server.go

```diff
@@ -65,6 +65,8 @@ type Config struct {
 	H2C                       bool          `mapstructure:"h2c"`
 	RandomDelay               bool          `mapstructure:"random-delay"`
 	RandomError               bool          `mapstructure:"random-error"`
+	Unhealthy                 bool          `mapstructure:"unhealthy"`
+	Unready 				  bool          `mapstructure:"unready"`
```

Comment:
Can you please run go fmt

Reply:
Done, need to look into my IDE settings as this happens all the time since the reinstall of my laptop.